pkg/applicationserver/io/packages/alcsync/v1/alcsync.go

```diff
@@ -60,27 +65,36 @@ func newTimeSyncCommand(
 }
 
 // MakeCommands parses the uplink payload and returns the commands.
-func MakeCommands(up *ttnpb.ApplicationUplink, fPort uint32, data *packageData) ([]Command, error) {
-	cID, cPayload := up.FrmPayload[0], up.FrmPayload[1:]
+func MakeCommands(up *ttnpb.ApplicationUplink, fPort uint32, data *packageData) ([]Command, []events.Builder, error) {
+	cID, cPayload := ttnpb.ALCSyncCommandIdentifier(up.FrmPayload[0]), up.FrmPayload[1:]
 	commands := make([]Command, 0)
+	evts := make([]events.Builder, 0)
 	for {
 		cmd, rest, err := makeCommand(cID, cPayload, up, fPort, data)
 		if err != nil {
-			return commands, err
+			err := errCommandCreationFailed.WithCause(err).WithAttributes(
```

Comment:
These attributes won't be visible anywhere as they are not part of the error message or public attributes.

If you would like to make them part of the error, you need to add them as part of the error definition. In effect, `errors.Define("name", "message", "public_attribute_1", "public_attribute_2")` and so forth.

Very nit: I don't think `up.ReceivedAt` is relevant to this error.